### ISSUE TITLE
paymentRequest.create promise callbacks not working

### ISSUE THREAD

**Issue by 719media** [braintree/braintree-web#388]
### General information

* SDK version: 3.37.0
* Environment: Sandbox and Production
* Browser and OS Mac OS 10.13.6 Chrome 69

### Issue description

I just noticed that using braintree-web v3.37.0 doesn't fire `then()` or `catch()` on the braintree.paymentRequest.create calls

It works fine when using v3.36.0

```
import braintreeClient from 'braintree-web/client';
import braintreePaymentRequest from 'braintree-web/payment-request';

braintreeClient.create({
    authorization: token
})
  .then(client => {
    //i get here
    braintreePaymentRequest.create({
      client: client
    })
      .then(instance => {
        //never gets here
      })
      .catch(() => {
        //never gets here
      });
  });
```

I will do some more digging to see if I can see what's happening

**Comment by 719media** [comment]
From what I can tell, the FRAME_READY event no longer is being hit for some reason

https://github.com/braintree/braintree-web/blob/882ac0adcf1798b8920e5094fbb24a331787a2b7/src/payment-request/external/payment-request.js#L214

**Comment by 719media** [comment]
Found the issue here:

https://github.com/braintree/braintree-web/blob/3.36.0/src/payment-request/external/payment-request.js#L112-L125

When this gets generated into /dist as part of the npm install, you wind up with

```
function composeUrl(assetsUrl, componentId, isDebug) {
  var baseUrl = assetsUrl;

  {
    // Pay with Google cannot tokenize in our dev environment
    // so in development, we have to use a sandbox merchant
    // but set the iFrame url to the development url
    baseUrl = 'https://' + undefined + ':9000';
  }
  // endRemoveIf(production)

  return baseUrl + '/web/' + VERSION + '/html/payment-request-frame' + useMin(isDebug) + '.html#' + componentId;
}
```

which is obviously incorrect

**Comment by crookedneighbor** [comment]
Thanks for the report, we'll get this fixed up.

**Comment by crookedneighbor** [comment]
We've pinpointed the problem. Aiming to get a fix out for this on Monday, 9/24

**Comment by crookedneighbor** [comment]
This should be fixed in 3.38.0. Thanks for letting us know!
